test/test_helper.exs

```diff
@@ -1 +1,3 @@
+{:ok, _} = Application.ensure_all_started(:telemetry)
```

Comment:
Just for future reference, why was this necessary?

Reply:
`Ex.Aws` has telemetry turned on, and it just vomits errors to my screen if it is not started.